main.tf

```diff
@@ -245,3 +245,12 @@ module "nomad" {
   # Redis
   redis_port = var.redis_port
 }
+
+
+module "grafana" {
```

Comment:
Let's keep it just as a separate module, it's only optional module and it can become very annoying

Reply:
okay, then maybe in the makefile have it be like `make plan-grafana` and `make apply-grafana`
 ?